### PR TITLE
refactor: factor out showDeleteSpace listener

### PR DESCRIPTION
**Pull request by fboechats**
Factor out the showDeleteSpace listener to its own file.

refs #104

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
Hi @fboechats, will you be able to incorporate the changes? Thanks!

**Comment by juancarlosfarah** [comment]
Hi @fboechats, thanks for your changes! Could you please change the file name and squash all of the commits into one? With the message `refactor: factor out showDeleteSpacePrompt listener`? That way your commit gets included in our CHANGELOG correctly, as it will be following conventional commit format. Sorry for the hassle. I would do it, but I don't think I can right to the branch on your repo. Thanks!

File: public/app/listeners/showDeleteSpace.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const { dialog } = require('electron');
 const { RESPOND_DELETE_SPACE_PROMPT_CHANNEL } = require('../config/channels');
 
-const showDeleteSpace = mainWindow => () => {
+const showDeleteSpacePrompt = mainWindow => () => {
```
Comment (juancarlosfarah) [inline public/app/listeners/showDeleteSpace.js]: This file should also be called `showDeleteSpacePrompt` for consistency.

File: public/app/listeners/showExportSpace.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 const { dialog } = require('electron');
 const { RESPOND_EXPORT_SPACE_PROMPT_CHANNEL } = require('../config/channels');
 
-const showExportSpace = mainWindow => (event, spaceTitle) => {
+const showExportSpacePrompt = mainWindow => (event, spaceTitle) => {
```
Comment (juancarlosfarah) [inline public/app/listeners/showExportSpace.js]: This file should also be called `showExportSpacePrompt` for consistency.

File: public/app/listeners/showLoadSpace.js [hunk 3]
```diff
@@ -2,10 +2,10 @@
 const { dialog } = require('electron');
 const { RESPOND_LOAD_SPACE_PROMPT_CHANNEL } = require('../config/channels');
 
-const showLoadSpace = mainWindow => (event, options) => {
+const showLoadSpacePrompt = mainWindow => (event, options) => {
```
Comment (juancarlosfarah) [inline public/app/listeners/showLoadSpace.js]: This file should also be called `showLoadSpacePrompt` for consistency.

**Comment by juancarlosfarah** [comment]
Hi @fboechats, just wanted to know if you were going to be able to address these changes / rebase, otherwise I can try and make them myself in order to go forward with these PRs. Thanks!

**Comment by fboechats** [comment]
Hi @juancarlosfarah , sorry for the delay again, just did the changes you asked.

**Comment by juancarlosfarah** [comment]
@fboechats, would it be possible for you to squash all the 12 commits into one? That way the commit history stays clean.

Here is how to do it in case you haven't done it before (subtitle: The hard(er) and less flexible way):

https://gist.github.com/patik/b8a9dc5cd356f9f6f980#the-harder-and-less-flexible-way

The commit message can then be similar to the first one:

```
refactor: factor out showLoadSpacePrompt listener

Factor out the showLoadSpacePrompt listener to its own file.

refs #104
```

Many thanks!

**Comment by fboechats** [comment]
Hi @juancarlosfarah just squashed the commits, after we finished this I will keep the refactor

**Comment by juancarlosfarah** [comment]
Thanks @fboechats! I'm doing the merge manually in 50d2eb9d48fa6d10791010c044d2fbe309e965bc and 3b44fd7a67ea34c47f64bdb0be3cfe6cc4e0ac66 to resolve conflicts and will close this and the other PRs.

**Pull request merged**